master/buildbot/reporters/github.py

```diff
@@ -171,19 +171,24 @@ def send(self, build):
                     issue=issue,
                     description=description
                 )
+
+                assert response.code // 100 == 2
```

Comment:
Please do not use assert. assert is supposed to be disabled in production.

Also // 100 == 2 is a bit hard to read

Reply:
Which method do you prefer to check the status code? (I personally always use `//`

Reply:
I would create a helper method isStatus2XX(response)

Reply:
ha, okay.